Failure to startup with io.quarkus:quarkus-smallrye-reactive-messaging-kafka extensions

Error during startup:

`Caused by: javax.enterprise.inject.spi.DefinitionException: Normal scoped beans must declare a non-private constructor with no parameters: CLASS bean [types=[io.smallrye.reactive.messaging.impl.StreamFactoryImpl, java.lang.Object, io.smallrye.reactive.messaging.StreamFactory], qualifiers=[@Default, @Any], target=io.smallrye.reactive.messaging.impl.StreamFactoryImpl]`

To replicate create a standard hello rest service. Add fault-tollerance and messaging kafka

`mvn quarkus:add-extension -Dextensions=io.quarkus:quarkus-smallrye-fault-tolerance,io.quarkus:quarkus-smallrye-reactive-messaging-kafka`

Simply startup dev

`mvn compile quarkus:dev`

--- a/bom/pom.xml
+++ b/bom/pom.xml
@@ -53,7 +53,7 @@
         <smallrye-jwt.version>1.1.4</smallrye-jwt.version>
         <smallrye-reactive-streams-operators.version>1.0.3</smallrye-reactive-streams-operators.version>
         <smallrye-converter-api.version>1.0.3</smallrye-converter-api.version>
-        <smallrye-reactive-messaging.version>0.0.7</smallrye-reactive-messaging.version>
+        <smallrye-reactive-messaging.version>0.0.8</smallrye-reactive-messaging.version>
         <smallrye-rest-client.version>1.2.2</smallrye-rest-client.version>
         <swagger-ui.version>3.20.9</swagger-ui.version>
         <javax.activation.version>1.1.1</javax.activation.version>

